e2e/fixtures.ts

```diff
@@ -49,3 +64,63 @@ export async function argosScreenshot(
     });
   }
 }
+
+/**
+ * Load the app into `page` and send `entities` to the app to be loaded into the
+ * database.
+ *
+ * If `entities` is undefined, the default demo data is used.
+ */
+export async function loadApp(page: Page, entities?: Entity[]) {
+  if (entities) {
+    await page.addInitScript((data) => {
+      window["ndbDemoData"] = data;
+    }, serializeEntities(entities));
+  }
+
+  await page.goto("/?useCase=education");
+
+  await page.getByText("Start Exploring").click({ timeout: 10_000 });
+}
+
+/**
+ * Take a collection of entities and serialize them using an
+ * `EntitySchemaService`.
+ *
+ * This implementation is not fully compatible with serialization in the app.
+ */
+function serializeEntities(entities: Entity[]): unknown[] {
+  const entitySchemaService = new EntitySchemaService({
+    // Mock an injector that can only retrieve `DefaultDatatype`.
+    get(type: unknown) {
+      assert(type === DefaultDatatype);
+      return [
+        new DefaultDatatype(),
+        new StringDatatype(),
+        new DateWithAgeDatatype(),
+        new ConfigurableEnumDatatype(null),
+        new DateOnlyDatatype(),
+        new LocationDatatype(null),
+        new EntityDatatype(null, null, null),
+        new EventAttendanceMapDatatype(entitySchemaService),
+        new LongTextDatatype(),
+      ] satisfies DefaultDatatype[];
+    },
+  });
+
+  const config = new Config(Config.CONFIG_KEY, defaultJsonConfig);
+  const entityConfigService = new EntityConfigService(null, entityRegistry);
+  entityConfigService.setupEntities(
+    Object.entries(config.data.data).flatMap(([id, config]) => {
```

Comment:
```suggestion
  const entityConfigService = new EntityConfigService(null, entityRegistry);
  entityConfigService.setupEntities(
    Object.entries(defaultJsonConfig.data).flatMap(([id, config]) => {
```

Could possibly simplify it like this?